pkg/blocks/basicstep.go

```diff
@@ -172,21 +93,13 @@ func (b *BasicStep) Validate(execCtx TTPExecutionContext) error {
 		return err
 	}
 
-	// Validate CleanupStep if it is not nil
-	if b.CleanupStep != nil {
-		if err := b.CleanupStep.Validate(execCtx); err != nil {
-			logging.L().Errorw("error validating cleanup step", zap.Error(err))
-			return err
-		}
-	}
-
 	logging.L().Debugw("command found in path", "executor", b.Executor)
 
 	return nil
 }
 
 // Execute runs the BasicStep and returns an error if any occur.
-func (b *BasicStep) Execute(execCtx TTPExecutionContext) (*ExecutionResult, error) {
```

Comment:
These now return `ActResult` because `ExecutionResult` is only for actions that are part of a step, but `Execute` is also called by cleanup actions - this is an in the weeds thing, but trust me it makes sense